code/datums/body_archive.dm

```diff
@@ -115,7 +115,32 @@ var/list/body_archives = list()
 	archive.data["dna_records"] = R
 	archive.data["underwear"] = underwear
 
-	//I gave a shot at preserving limb status (destroyed, peg, mechanized) then decided I was too tired and it's too much trouble, so TODO for anyone with the determination
```

Comment:
There's something to be learned here but I'm not sure what it was.

Reply:
not coding while tired?

Reply:
maybe that yeah